### ISSUE TITLE
Firefly-vis 2.0.3 breaks yt

### ISSUE THREAD

**Issue by neutrinoceros** [ageller/Firefly#62]
firefly-vis' integration in yt relies on `Firefly.data_reader.Reader` to be available. Apparently the `data_reader` was removed (or moved), so this is showing up as a breaking change there.

Would you consider resurecting the `data_reader` module, be it as a pure backward compatibility layer where you'd just import stuff from other places so it can still be found there from the outside world ?

**Comment by neutrinoceros** [comment]
oh I see now that the module/class are in fact still where they're supposed to be, there must have been something wrong with packaging that release (in which case, please consider yanking it from PyPI).
Unfortunately there is no v2.0.3 tag so it's harder to tell what exact commit this version number maps to, and when/if the breaking change happened on the repo.

**Comment by agurvich** [comment]
Sorry... I'm still getting the hang of managing releases. The issue was a totally separate package of mine that had a dumb import error. I've fixed that package, deployed it, and I'm able to import Firefly.data_reader.Reader now.

**Comment by agurvich** [comment]
I think Firefly-vis 2.0.3 should be working now so long as abg_python (a new dependency I added to Firefly-vis in 2.0.3) is updated to 1.0.2 (which I just deployed). When I tried to import `Firefly.data_reader.Reader` I was getting an import error in `abg_python.snapshot_utils` (which Firefly-vis uses to open FIRE .hdf5 files) for `abg_python.all_utils` (which doesn't exist as of 1.0.2, but did in 1.0.1). 

Of course locally I had abg_python in an up-to-date unpublished state and so I didn't notice this problem in abg_python until this morning when you pointed it out. 

If `from Firefly.data_reader import Reader` still doesn't work can you point me to the error message you're getting?

**Comment by neutrinoceros** [comment]
ah ! indeed updating abg_python fixes the issue, but this still means that Firefly-vis should be patched to specify this requirement.
I'll open a PR.

**Comment by agurvich** [comment]
good point, i guess in the setup.py it should have a minimum version requirement?
